.github/workflows/puppet-test.yml

```
@@ -20,6 +20,8 @@ on:
       - 'internal/buildscripts/packaging/tests/helpers/**'
       - 'internal/buildscripts/packaging/tests/requirements.txt'
       - '!**.md'
+  schedule:
+      - cron: '0 0 * * 3,6' # Every Wednesday and Saturday at midnight UTC
```

```suggestion
  schedule:
    - cron: '0 0 * * 3,6' # Every Wednesday and Saturday at midnight UTC
```
just for consistency 😅